docs/en_US/TrainingService/PaiMode.md

```diff
@@ -32,8 +32,6 @@ trial:
   cpuNum: 1
   memoryMB: 8196
   image: msranni/nni:latest
```

Comment:
so I want to output my models, where is the output path?

Reply:
PAI does not have `dataDir` and `outputDir` in their doc anymore, so we remove these two path accordingly. In other training modes, users use `OUTPUTDIR` in trial code to output the data they want, @SparkSnail could you check the content of `OUTPUTDIR` in PAI mode?

Reply:
any doc to tell?

Reply:
Users could use `PAI_OUTPUT_DIR` to get the output directory of PAI, the defaulet value is $PAI_DEFAULT_FS_URI/Output/$jobName, refer https://github.com/microsoft/pai/blob/b2324866d0280a2d22958717ea6025740f71b9f0/docs/job_tutorial.md